content/en/cd-as-a-service/release-notes/known-issues.md

```diff
@@ -3,10 +3,6 @@ title: Known Issues and Limitations
 
 ---
 
-## Strategy to use with a HorizontalPodAutoscaler
```

Comment:
Good catch - I'd run removal past Parth or Dan.

Reply:
removed this